frontend/model/contracts/chatroom.js

```diff
@@ -213,14 +213,20 @@ sbp('chelonia/defineContract', {
         if (!memberID) {
           throw new Error('The new member must be given either explicitly or implcitly with an inner signature')
         }
+
         if (!state.shouldSaveMessages) {
```

Comment:
Still unsure about the name for this field, but I also can't think of a much better name. What about keeping the old one? The issue with this name is that:

* It saves all messages, but the name implies that messages aren't saved if not set. This is incorrect, as some messages are still saved (hence this PR)
* Besides saving messages, it skips some validation, which is needed for rendering, but which means that it shouldn't be used in other contexts.

If you don't like the previous name, what about something like `renderingContext`, along with a comment explaining what it does and why the rendering code needs this different logic?

Reply:
What do you think, @taoeffect? If you are fine with the `renderingContext`, let's go with that.

Reply:
@Silver-IT I actually like `renderingContext` a lot! It seems like a great name for this 👍